weather_sp/README.md

```diff
@@ -51,12 +54,28 @@ Using DataflowRunner
 ```bash
 weather-sp --input-pattern 'gs://test-tmp/era5/2015/**' \
            --output-dir 'gs://test-tmp/era5/splits'
-           --formatting '.{typeOfLevel}'
+           --formatting '.{typeOfLevel}' \
+           --runner DataflowRunner \
+           --project $PROJECT \
+           --temp_location gs://$BUCKET/tmp  \
+           --job_name $JOB_NAME
+```
+
+Using ecCodes-powered grib splitting on Dataflow (this is often more robust, especially when splitting multiple 
```

Comment:
Can you add an explanation of what the `sdk_contrainer_image` needs to contain, e.g. eccodes, don't know if anything else.

Reply:
Makes sense. Added.